internal/databases/postgres/tar_ball_composer.go

```diff
@@ -57,6 +62,21 @@ func NewTarBallComposerMaker(composerType TarBallComposerType, conn *pgx.Conn,
 			return nil, err
 		}
 		return NewRatingTarBallComposerMaker(relFileStats, filePackOptions)
+	case CopyComposer:
+		previousBackupName, err := internal.GetLatestBackupName(folder)
+		if err != nil {
+			return NewRegularTarBallComposerMaker(filePackOptions), nil
+		}
+		tracelog.ErrorLogger.PanicOnError(err)
+		previousBackup := NewBackup(folder, previousBackupName)
+		prevBackupSentinelDto, err := previousBackup.GetSentinel()
+		tracelog.ErrorLogger.PanicOnError(err)
```

Comment:
Again, all errors should be handled on the top level, just return them, do not panic :)